deploy/awstools/awstools.py

```diff
@@ -261,6 +278,17 @@ def launch_instances(instancetype, count, instancemarket, spotinterruptionbehavi
         if len(instances) < count:
             rootLogger.info("Launching remaining {} {} instances".format(count - len(instances), instancetype))
 
+    # append the tags from the manager to the launched instance
+    extra_tags = get_manager_tags()
+    # filter out the 'Name'
+    extra_tags.pop('Name', None)
+    # filter out any tags with the word 'manager' (for CI)
```

Comment:
Re: these commends. I think the code, which is relatively straightforward, explains clearly _what_ is happening here -- it would be better to explain _why_ in the comment by expanding the parenthetical `(for CI)` to explain, for instance, what would happen if you didn't filter these out.

Reply:
Better?